.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,29 @@
+name: Feature Request
+description: A feature request for the Service Catalog
+title: <your-feature-request-title>
+body:
+- type: textarea
+  id: context
+  attributes:
+    description: |
+      Please add any additional context for your story.
+    label: Additional context
+  validations:
+    required: true
+- type: textarea
+  id: criteria
+  attributes:
+    label: Acceptance Criteria
+    placeholder: |
+      - [ ] At least once acceptance criteria should be present
+  validations:
+    required: true
+- type: textarea
+  id: links
+  attributes:
+    label: Relevant links
+    description: |
+      Add any links to resources, issues, etc. that you feel are
+      relevant to this story.
+  validations:
+    required: true
```

Comment:
Why are links required here?

Reply:
Leftover validation from different issue, removed it :+1: